app/src/main/java/org/fossasia/susi/ai/skills/groupwiseskills/GroupWiseSkillsFragment.kt

```diff
@@ -43,6 +44,7 @@ class GroupWiseSkillsFragment : Fragment(), IGroupWiseSkillsView, SwipeRefreshLa
         val argument = arguments
         if (argument != null) {
             this.skills.group = argument.getString("group")
+            (activity as SkillsActivity).setActionBarTitle(this.skills.group)
```

Comment:
And if in future, `GroupWiseSkillsFragment.kt` is attached to a different Activity?

This is type unsafe and will lead to problems in future